documents/model.go

```diff
@@ -53,6 +69,27 @@ const (
 	ErrZeroCollaborators = errors.Error("require at least one collaborator")
 )
 
+// NewDefaultTree returns a DocumentTree with default opts
+func NewDefaultTree(salts *proofs.Salts) *proofs.DocumentTree {
+	return NewDefaultTreeWithPrefix(salts, "")
+}
+
+// NewDefaultTreeWithPrefix returns a DocumentTree with default opts passing a prefix to the tree leaves
+func NewDefaultTreeWithPrefix(salts *proofs.Salts, prefix string) *proofs.DocumentTree {
+	var prop proofs.Property
+	if prefix != "" {
+		prop = proofs.NewProperty(prefix)
+	}
+
+	t := proofs.NewDocumentTree(proofs.TreeOptions{CompactProperties: true, EnableHashSorting: true, Hash: sha256.New(), ParentPrefix: prop, Salts: salts})
+	return &t
+}
+
+// NewLeafProperty returns a proof property with the literal and the compact
+func NewLeafProperty(literal string, compact []byte) proofs.Property {
```

Comment:
Same for creating custom properties, I wanted to enforce that we always pass the compact name as well